spyder/plugins/ipythonconsole/widgets/namespacebrowser.py

```diff
@@ -70,6 +70,12 @@ def get_value(self, name):
             "it's not installed alongside Spyder. To fix this problem, please "
             "install it in the same environment that you use to run Spyder."
         )
+        reason_mismatched_numpy = _(
+            "There is a mismatch between the Numpy versions used by Spyder "
+            "and the kernel of your current console. To fix this problem, "
+            "please upgrade `numpy` in the environment that you use to run "
+            "Spyder to version 1.26.1 or higher."
```

Comment:
```suggestion
            "please upgrade <tt>numpy</tt> in the environment that you use to "
            "run Spyder to version 1.26.1 or higher."
```

Since Qt can't render Markdown, we need to use the `tt` tag to `numpy` appear in a monospace font.